server-webrtc/http.go

```diff
@@ -72,3 +73,49 @@ func receiveSignalsHTTP(addr string, config *webrtc.Configuration) error {
 	log.Printf("listening HTTP on %s", addr)
 	return http.ListenAndServe(addr, nil)
 }
+
+func deserializeSessionDescription(msg string) *webrtc.SessionDescription {
```

Comment:
This code will now live in three places: `client/lib/util.go`, `proxy-go/snowflake.go`, and here.  Maybe it belongs under `common/`?

Reply:
Hm, yeah. I'd say we should definitely do this if we're going to maintain server-webrtc. If not, then we could still do it but I'd prefer to make that a separate refactor ticket.

Reply:
Sounds good

Reply:
I took a stab at this here,
https://github.com/keroserene/snowflake/commit/7b761d4c8d0e56b9148f106eb01667a7ec5c0424